src/api/index.js

```diff
@@ -14,33 +15,33 @@ const getExampleData = () => {
     .then(response => response.data);
 };
 
-const getAuthHeader = authState => {
-  if (!authState.isAuthenticated) {
-    throw new Error('Not authenticated');
-  }
-  return { Authorization: `Bearer ${authState.idToken}` };
-};
+// const getAuthHeader = authState => {
+//   if (!authState.isAuthenticated) {
+//     throw new Error('Not authenticated');
+//   }
+//   return 'hi';
 
-const getDSData = (url, authState) => {
+// };
```

Comment:
The commented out code should be deleted, unless it is intended to be used in the future. If that is the case, please write a comment explaining that.

Reply:
Good catch